app/controllers/affiliations_controller.rb

```diff
@@ -0,0 +1,24 @@
+class AffiliationsController < ApplicationController
+  # POST /affiliations or /affiliations.json
+  def create
+    @affiliation = Affiliation.new(politician_id: affiliation_params[:politician_id], relationship: affiliation_params[:relationship], business_id: affiliation_params[:business_attributes][:id]) if affiliation_params[:business_attributes] && affiliation_params[:business_attributes][:id].present?
```

Comment:
You can wrap this in an if-else for easier readability

Also, not sure why we have the two ifs considering an affiliation belongs to a business, therefore, the business_id will always have to exist